programs/fileio.c

```diff
@@ -672,7 +672,7 @@ static unsigned long long FIO_compressLz4Frame(cRess_t* ress,
     prefs.compressionLevel = compressionLevel;
     prefs.frameInfo.blockMode = LZ4F_blockIndependent; /* stick to defaults for lz4 cli */
     prefs.frameInfo.blockSizeID = LZ4F_max4MB;
-    prefs.frameInfo.contentChecksumFlag = (contentChecksum_t)g_checksumFlag;
+    prefs.frameInfo.contentChecksumFlag = (LZ4F_contentChecksum_t)g_checksumFlag;
```

Comment:
Is `LZ4F_contentChecksum_t` available in all versions of lz4 that have frame support? I seem to remember using `contentChecksum_t` because that isn't the case.

Reply:
The `LZ4F_` prefix dates from [this commit](https://github.com/lz4/lz4/commit/081bcca33bba4f6ac3e913b3cf2bc2ba58b6b697) in mid-2015. So no, I guess not. When I built, though, `LZ4F_ENABLE_OBSOLETE_ENUMS` wasn't set, so the build failed. The alternative is to set `LZ4F_ENABLE_OBSOLETE_ENUMS` before I include `lz4frame.h`, I guess.

Reply:
Yeah, we should define that. It looks like it was changed in the last release https://github.com/lz4/lz4/commit/4531637ecdc4b12154c66cc69cfaade185039e4c.